commit 728c1ace30f2338b86331f0b83559409198e1f74
Author: Taylor Hornby <havoc@defuse.ca>
Date:   Wed Sep 9 16:46:48 2015 -0600

    g++-4.9 requires commas to include the argument to ``rpath`` as a linker flag.

diff --git a/Makefile b/Makefile
index 5f59388..68e91bb 100644
--- a/Makefile
+++ b/Makefile
@@ -18,7 +18,7 @@ DEPSRC = depsrc
 DEPINST = depinst
 
 CXXFLAGS += -I$(DEPINST)/include -Isrc
-LDFLAGS += -L$(DEPINST)/lib -Wl,-rpath $(DEPINST)/lib
+LDFLAGS += -L$(DEPINST)/lib -Wl,-rpath,$(DEPINST)/lib
 LDLIBS += -lgmpxx -lgmp -lboost_program_options
 # OpenSSL and its dependencies (needed explicitly for static builds):
 LDLIBS += -lcrypto -ldl -lz
